### PR TITLE
Quo - truncation for component wallet/account card fix truncation

### LINKED ISSUE #18044: Quo - fix longname for component wallet/account card
If the name is too long on a wallet account it should be an ellipsis at the end of the name to make it sit right.

 The account name should stay on one line and end with ellipsis (...)
Screenshot 2023-12-01 at 12 31 08

Here's an example:
https://www.figma.com/file/HKncH4wnDwZDAhc4AryK8Y/Wallet-for-Mobile?type=design&node-id=7014%3A65125&mode=design&t=AYBw5yLYHfVclhgo-1

### PR DESCRIPTION
**Pull request by Pau1fitz**
Quo - If the name is too long on a wallet account text should be truncated using an ellipsis.

Fixes #18044  

[Figma example](https://www.figma.com/file/HKncH4wnDwZDAhc4AryK8Y/Wallet-for-Mobile?type=design&node-id=7014%3A65125&mode=design&t=AYBw5yLYHfVclhgo-1)

Before:
Screenshot 2023-12-21 at 19 09 56

After:
Screenshot 2023-12-20 at 14 10 16

Testing notes:
Create a new wallet account with a really long name
Go to wallet home page
Observe wallet card uses ellipsis as in the designs.

Can also be tested in the Quo preview screen for wallet -> account card component

Can be tested on release builds here:
https://github.com/status-im/status-mobile/pull/18256#issuecomment-1864662289

### CODE REVIEW

File: src/quo/components/wallet/account_card/view.cljs [hunk 1]
```diff
@@ -128,6 +128,8 @@
              [text/text
               {:size   :paragraph-2
                :weight :medium
+               :number-of-lines 1
+               :ellipis-mode "tail"
```
Comment (J-Son89) [inline src/quo/components/wallet/account_card/view.cljs]: most likely can use a keyword, i.e 
`:ellipis-mode :tail`

Comment (Pau1fitz) [reply src/quo/components/wallet/account_card/view.cljs]: updated 👍

**Review by siddarthkay — APPROVED** [review]
looks good

**Review by smohamedjavid — APPROVED** [review]
Thank you for the fix! @Pau1fitz 🚀

**Review by J-Son89 — APPROVED** [review]

**Review by OmarBasem — APPROVED** [review]

**Comment by J-Son89** [comment]
@status-im/mobile-qa - this can be tested here: https://github.com/status-im/status-mobile/pull/18259

**Comment by Pau1fitz** [comment]
@smohamedjavid @OmarBasem @siddarthkay  @J-Son89 

is the below a known issue? When you type the keyboard doesn't expand.


https://github.com/status-im/status-mobile/assets/6873736/9f1d1824-a98a-4d3c-b8e0-89fc7ef1bf10

**Comment by smohamedjavid** [comment]
> is the below a known issue? When you type the keyboard doesn't expand.



It's the iOS simulator's behaviour when you type using your system (hard) keyboard. The device's (soft) keyboard can be toggled by using `cmd (⌘) + K`.

**Comment by pavloburykh** [comment]
@Pau1fitz thanx for the PR. 

In case of watch only account we need to take into consideration the size of  the "eye icon". Otherwise in case of long name it is displaced too close to the wallet card border (see screenshot below). @J-Son89 WDYT? I have not found an example of long name watch only account in Figma. 

Below screenshot is taken on iPhone X, IOS 16.4

photo_2023-12-21 18 54 17

**Comment by J-Son89** [comment]
> @Pau1fitz thanx for the PR.
> 
> In case of watch only account we need to take into consideration the size of the "eye icon". Otherwise in case of long name it is displaced too close to the wallet card border (see screenshot below). @J-Son89 WDYT? I have not found an example of long name watch only account in Figma.
> 
> Below screenshot is taken on iPhone X, IOS 16.4
> 
> photo_2023-12-21 18 54 17

good spot @Pau1fitz, yeah should be easy enough to add this. Seems just like some padding or margin should sort it 👍

**Comment by Pau1fitz** [comment]
after pulling the latest code and running `make run-metro` I am seeing the following error. Has anyone seen this before?

```
 ERROR  TypeError: native_module.core.status.call(null).fleets is not a function. (In 'native_module.core.status.call(null).fleets()', 'native_module.core.status.call(null).fleets' is undefined)
```

**Comment by J-Son89** [comment]
ah yes, there was a recent breaking change made to Status-Go where fleets changed. You will have to re run `make run-ios` to rebuild the app

**Comment by Pau1fitz** [comment]
@pavloburykh ready for re-testing. See screenshots below


Screenshot 2023-12-21 at 19 04 02

Screenshot 2023-12-21 at 19 05 41

**Comment by pavloburykh** [comment]
@J-Son89 could you please update correspondent https://github.com/status-im/status-mobile/pull/18259 so I can test changes? Thank you.

**Comment by pavloburykh** [comment]
@J-Son89 nevermind. Sorry, I have forgotten that you are off.

**Comment by J-Son89** [comment]
No problem @pavloburykh, I had one or two questions about the update I sent to @Pau1fitz. Once I get an answer I'll update the other pr with changes. We can wait with testing until then. Thanks!

**Comment by pavloburykh** [comment]
@Pau1fitz thank you for the fix. Tested in https://github.com/status-im/status-mobile/pull/18256. Ready for merge.

**Comment by pavloburykh** [comment]
> No problem @pavloburykh, I had one or two questions about the update I sent to @Pau1fitz. Once I get an answer I'll update the other pr with changes. We can wait with testing until then. Thanks!

Oh, okay. Missed this comment @J-Son89 and moved PR to merged already. Ping me up if it will require re-test.

**Comment by J-Son89** [comment]
Ah no problem, good from my side now too so I'll merge it 👌

**Pull request closed without merging**